packages/now-cli/src/commands/deploy/legacy.ts

```diff
@@ -304,6 +304,16 @@ export default async function main(
     `You are using an old version of the Now Platform. More: ${link(infoUrl)}`
   );
 
+  if (argv.prod) {
+    error(
+      `Now 1.0 does not support Production Deployments. Please use Now 2.0 to take advantage this feature: ${link(
+        'https://zeit.co/upgrade'
```

Comment:
Should we also tell the user how to achieve what they want for the 1.0 deployment? (i.e. manual `now alias` or whatever?)

Reply:
Let's do that 👍

Reply:
> Option --prod is not supported for Now 1.0 deployments. To manually alias a deployment, use now alias instead.
What do you think @leo @TooTallNate ?

Reply:
That's great! But let's make `now alias` use a code snippet.